reports-backend/src/main/java/org/jboss/da/listings/api/model/ProductArtifactRelationship.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.da.listings.api.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+@ToString
+public class ProductArtifactRelationship {
```

Comment:
You want to have Product field in artifact right? And if you have one artifact in multiple products?